Makefile

```diff
@@ -1,5 +1,5 @@
 TOOL?=vault-plugin-auth-kubernetes
-TEST?=$$(go list ./... | grep -v /vendor/)
+TEST?=$$(go list ./...)
```

Comment:
Looks like we could remove the `| grep -v vendor` from GOFMT_FILES too? 

https://github.com/hashicorp/vault-plugin-auth-kubernetes/blob/e108b81f10613b52866cc9f7659b94d3154c481a/Makefile#L8

Reply:
Good  catch, missed that one.

Reply:
Fixed in https://github.com/hashicorp/vault-plugin-auth-kubernetes/pull/115/commits/441d1b4301a351fc8f3b0c73eb5746ff4553a791